### PR TITLE
feat: Add node pool parameter

### PR DESCRIPTION
**Pull request by Vafilor**
**What this PR does**:

**Which issue(s) this PR fixes**:

Fixes onepanelio/core#

**Special notes for your reviewer**:

**Checklist**

Please check if applies

- [ ] I have added/updated relevant unit tests
- [ ] I have added/updated relevant documentation

Required 

- [x] I accept to release these changes under the Apache 2.0 License

### CODE REVIEW

File: db/yaml/maskrcnn/20201221195937.yaml [hunk 1]
```diff
@@ -124,34 +116,35 @@ templates:
...
-        image: tensorflow/tensorflow:2.3.0
-        command: [sh, -c]
-        env:
-          - name: ONEPANEL_INTERACTIVE_SIDECAR
-            value: 'true'
-        args: ["tensorboard --logdir /mnt/output/"]
-        ports:
-          - containerPort: 6006
-            name: tensorboard
+      sidecars:
```
Comment (rushtehrani) [inline db/yaml/maskrcnn/20201221195937.yaml:119]: This indentation seems incorrect - it should line up with `nodeSelector`

File: db/yaml/maskrcnn/20201221195937.yaml [hunk 2]
```diff
@@ -11,13 +11,13 @@ arguments:
       value: annotation-dump/sample_dataset
```
Comment (rushtehrani) [inline db/yaml/maskrcnn/20201221195937.yaml:11]: Bit confused about this file, are you deleting an old migration file here?

Comment (Vafilor) [reply db/yaml/maskrcnn/20201221195937.yaml:11]: @rushtehrani This migration adds the `select.nodepool` parameter

File: db/yaml/tfod/README.md [hunk 3]
```diff
@@ -0,0 +1 @@
+https://github.com/onepanelio/templates/blob/master/workflows/maskrcnn-training/template.yaml
```
Comment (rushtehrani) [inline db/yaml/tfod/README.md:1]: don't think we need this README here

**Comment by rushtehrani** [comment]
@Vafilor can you address the conflict?

**Comment by Vafilor** [comment]
> @Vafilor can you address the conflict?

@rushtehrani Sure thing - should be good.

**Pull request merged**